fast/stages/01-resman/organization.tf

```diff
@@ -93,13 +93,13 @@ module "organization" {
   }
   policy_list = {
     "constraints/cloudfunctions.allowedIngressSettings" = merge(
-      local.list_deny, { values = ["ALLOW_INTERNAL_ONLY"] }
+      local.list_deny, { values = ["is:ALLOW_INTERNAL_ONLY"] }
     )
     "constraints/cloudfunctions.allowedVpcConnectorEgressSettings" = merge(
-      local.list_deny, { values = ["PRIVATE_RANGES_ONLY"] }
+      local.list_deny, { values = ["is:PRIVATE_RANGES_ONLY"] }
     )
     "constraints/compute.restrictLoadBalancerCreationForTypes" = merge(
-      local.list_deny, { values = ["in:INTERNAL"] }
+      local.list_deny, { values = ["in:EXTERNAL"] }
```

Comment:
why the change to in:EXTERNAL?

Reply:
Goal is to deny the creation of XLBs and only allow ILBs.

Reply:
This has been replaced with the local syntax now